resources/warnings.json

```diff
@@ -17207,8 +17207,8 @@
     "url": "https://www.jenkins.io/security/advisory/2023-10-25/#SECURITY-2875",
     "versions": [
       {
-        "lastVersion": "1.0.9",
-        "pattern": ".*"
+        "lastVersion": "1.0.11",
+        "pattern": "(1[.]0[.]10|1[.][0].[0123456789])(|[.-].+)"
```

Comment:
```suggestion
        "pattern": "(1[.]0[.]10|1[.][0].[0-9])(|[.-].+)"
```

Both are correct, this is just more efficient (and IMO easier to read).

Reply:
Hi @daniel-beck,
Yes I had made this change locally indeed, I think I have reverted it by mistake. I will commit it.